opcua/ua/uatypes.py

```diff
@@ -1176,4 +1174,52 @@ def __str__(self):
     __repr__ = __str__
 
 
+# TODO: This should be autogeneratd form XML description of EventTypes
+class BaseEvent(FrozenClass):
+
+    '''
+    BaseEvent implements BaseEventType from which inherit all other events and it is used per default.
+
+    '''
 
+    def __init__(self, sourcenode=NodeId(ObjectIds.Server), message=None, severity=1):
```

Comment:
I agree BaseEvent is a good name but I think it should take a  type as argument so we can set all attributes dynamically.